### ISSUE TITLE
Emacs Change Color 

### ISSUE THREAD

**Issue by iKrishneel** [IonicaBizau/terminal-flat-theme#4]
I applied this theme on the Ubuntu Terminal.
I was wondering if in emacs i am using hl-line 
![screenshot from 2015-05-05 21 14 57](https://cloud.githubusercontent.com/assets/9315167/7472282/e65a0b82-f36b-11e4-8d05-11df43e28c32.png)

As you see this theme gives a color very light pinkish for color Pink. I want color Pink
Is there way I can fix this?

**Comment by IonicaBizau** [comment]
You should be able to change this in the terminal preferences.

Then you can export the xml file this way:

```sh
gconftool-2 --dump '/apps/gnome-terminal' > gnome-terminal-conf.xml
```

If you like, please make a PR with the updated color. :+1: 

![image](https://cloud.githubusercontent.com/assets/2864371/7472314/de0ffefa-f339-11e4-8719-a36651294abe.png)

**Comment by IonicaBizau** [comment]
Fixed via #5. :fireworks:
